File: chapters/14/node/connect.js
const MongoClient = require('mongodb').MongoClient;

// Replace with your MongoDB connection string and database name
const uri = 'mongodb://<username>:<password>@<host>/website';

MongoClient.connect(uri, (err, client) => {
  if (err) {
    console.log(err.stack);
    return;
  }

  console.log('Connected Successfully!');

  client.close();
});

File: chapters/14/python/connect.py
import pymongo

# Replace with your MongoDB connection string and database name
connection_string = "mongodb://<username>:<password>@<host>"

database_name = "website"

# Connect to the MongoDB
client = pymongo.MongoClient(connection_string)

# Set the database
database = client[database_name]

# Example query to test the connection
collection = database["cookbook"]

doc = collection.find_one()

print(doc)
